platform/solr/solr-xpath/pom.xml

```diff
@@ -104,5 +104,10 @@
             <artifactId>lux</artifactId>
             <version>${lux.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.sf.saxon</groupId>
+            <artifactId>Saxon-HE</artifactId>
+            <version>9.5.1-3</version>
```

Comment:
Does it actually require this ancient version of Saxon or can we upgrade it to the latest?